sdcm/cluster.py

```diff
@@ -5122,7 +5131,10 @@ def node_setup(self, node, verbose=False, **kwargs):  # pylint: disable=unused-a
             self.log.debug('Skip loader setup for using a prepared AMI')
             return
 
-        elif node.distro.is_debian10 or node.distro.is_debian11:
+        elif node.distro.is_debian12:
+            node.install_package(package_name='openjdk-17-jre openjdk-17-jre-headless')
```

Comment:
why do we need this installed ? now that temurin-11-jre is installed ?

Reply:
This is normal install and it's something I haven't removed - I'll remove it now.